modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -605,7 +832,7 @@ protected function render() {
 			$tabs_title_html .= "<div {$title_render_attributes}>";
 			$tabs_title_html .= "\t<span {$tab_icon_attributes}> {$icon_html}</span>";
 			$tabs_title_html .= "\t<span {$tab_icon_active_attributes}> {$icon_active_html}</span>";
-			$tabs_title_html .= "\t<span>{$tab_title}</span>";
+			$tabs_title_html .= "\t<span {$tab_title_text}>{$tab_title}</span>";
```

Comment:
Usecase: we need the be able to style the title text separately from the icons. 

I will change the classname.

Reply:
I don't see it changed

Reply:
I had changed it in one place only. 

In the next code push it will be correct. 

![image](https://user-images.githubusercontent.com/94352322/176647712-49d50014-af0d-49e2-a48f-23770f97998b.png)

![image](https://user-images.githubusercontent.com/94352322/176647754-5e9c1cec-64d6-4a4d-bc58-cb35fba1acdc.png)